Decred Wallet/Features/Custom Dialogs/AccountSelectorDialog/AccountSelectorDialog.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  AccountSelectorDialog.swift
+//  Decred Wallet
+//
+// Copyright (c) 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import UIKit
+import Dcrlibwallet
+
+typealias AccountSelectorDialogCallback = (_ selectedWalletId: Int, _ selectedAccount: DcrlibwalletAccount) -> Void
+
+class AccountSelectorDialog: UIViewController {
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var walletsTableView: UITableView!
+    @IBOutlet weak var walletsTableViewHeightConstraint: NSLayoutConstraint!
+    
+    private var dialogTitle: String!
+    private var callback: AccountSelectorDialogCallback!
+
+    var wallets = [Wallet]()
+    var selectedWallet: DcrlibwalletWallet?
+    var selectedAccount: DcrlibwalletAccount?
+
+    let accountCellRowHeight: CGFloat = 74
+    let walletHeaderSectionHeight: CGFloat = 36
+    
+    static func show(sender vc: UIViewController,
+                     title: String,
+                     selectedWallet: DcrlibwalletWallet?,
+                     selectedAccount: DcrlibwalletAccount?,
+                     callback: @escaping AccountSelectorDialogCallback) {
+
+        let dialog = AccountSelectorDialog.instantiate(from: .CustomDialogs)
+        dialog.dialogTitle = title
+        dialog.callback = callback
+        dialog.selectedWallet = selectedWallet
+        dialog.selectedAccount = selectedAccount
+        dialog.modalPresentationStyle = .pageSheet
+        vc.present(dialog, animated: true, completion: nil)
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.titleLabel.text = self.dialogTitle
+
+        self.walletsTableView.hideEmptyAndExtraRows()
+        self.walletsTableView.dataSource = self
+        self.walletsTableView.delegate = self
+        
+        //Without these two lines the grouped UITableView has unnecessary top and bottom padding
+        self.walletsTableView.tableHeaderView = UIView(frame: CGRect(x: 0, y: 0, width: 1, height: 0.1))
+        self.walletsTableView.tableFooterView = UIView(frame: CGRect(x: 0, y: 0, width: 1, height: 0.1))
```

Comment:
I spent a while trying to figure out why this issue exists in the first place and it turns out it's because of the call to `self.walletsTableView.hideEmptyAndExtraRows()`. There are no extra rows because we don't allow the table height grow bigger than the content height. Remove those 3 lines and we're almost good to go.